manubot/cite/isbn.py

```diff
@@ -16,6 +18,9 @@ class Handler_ISBN(Handler):
     def inspect(self, citekey):
         import isbnlib
 
+        isbnlib.config.setthreadstimeout(seconds=timeout_seconds)
+        isbnlib.config.seturlopentimeout(seconds=timeout_seconds)
```

Comment:
slight preference for us to define a helper like `import_isbnlib` such that we don't have to repeat this configuration 3 times.

Reply:
Is a helper preferable to using only a global import and config for the library at the top of the module?

This is an ISBN getting module anyway.

Reply:
> Is a helper preferable to using only a global import and config for the library at the top of the module?

Usually not, although in this case let's assume we added the lazy importing of isbnlib for a reason and keep it that way.

Reply:
@xihh87 did you want to switch this to use a helper so we can merge?